finality-aleph/src/network/substrate.rs

```diff
@@ -1,8 +1,16 @@
-use std::{collections::HashMap, iter, pin::Pin, sync::Arc};
+use core::fmt;
```

Comment:
uh, rather import the types, renamed if needed, and better from `std` rather than `core` for at least some consistency.

Unfortunately we are very inconsistent in how refer to all the types when implementing `Display`. >x

Reply:
fixed